build.sh

```diff
@@ -142,8 +142,9 @@ if hasArg --ptds; then
 fi
 
 # Append `-DFIND_RMM_CPP=ON` to CMAKE_ARGS unless a user specified the option.
+SKBUILD_EXTRA_CMAKE_ARGS="${EXTRA_CMAKE_ARGS}"
 if [[ "${EXTRA_CMAKE_ARGS}" != *"DFIND_RMM_CPP"* ]]; then
-    EXTRA_CMAKE_ARGS="${EXTRA_CMAKE_ARGS} -DFIND_RMM_CPP=ON"
+    SKBUILD_EXTRA_CMAKE_ARGS="${SKBUILD_EXTRA_CMAKE_ARGS} -DFIND_RMM_CPP=ON"
```

Comment:
Ah, this is a super helpful change. This ought to eliminate a warning about an unused `FIND_RMM_CPP` variable in the C++ builds. Let's remember to propagate this fix to other repos.

Reply:
Yeah that's annoyed me for a while, but didn't get around to fixing it. And yes, I'm doing the same thing in corresponding PRs to other repos.